securedrop_client/gui/__init__.py

```diff
@@ -195,3 +196,43 @@ def get_elided_text(self, full_text: str) -> str:
 
     def is_elided(self) -> bool:
         return self.elided
+
+
+class SecureQPlainTextEdit(QPlainTextEdit):
+    MAX_TEXT_WIDTH = 75
+    MAX_NATURAL_TEXT_WIDTH = 650
+    HEIGHT_BASE = 60
+    LINE_HEIGHT = 20
+
+    def __init__(self, text: str = '') -> None:
+        super().__init__()
+        self.setReadOnly(True)
+        self.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+        self.document().setTextWidth(self.MAX_TEXT_WIDTH)
+        self.height = self.HEIGHT_BASE
+        self.setPlainText(text)
+
+        # Disable copy/paste context menu
+        self.setContextMenuPolicy(Qt.NoContextMenu)
+
+    def setPlainText(self, text: str) -> None:
+        super().setPlainText(text)
+
+        total_line_count = 0
+        for block_num in range(0, self.blockCount()):
+            block = self.document().findBlockByNumber(block_num)
+            line_count = math.ceil(block.length() / self.document().idealWidth())
```

Comment:
I think this approach is a smart way to handle setting the height. In double checking this, I just noticed that it underestimates the number of lines (at least for me locally), e.g. I have a `line_count` of 4 for the following block which is actually taking up 5 lines in the UI:

![Screen Shot 2020-04-17 at 11 54 51 AM](https://user-images.githubusercontent.com/7832803/79588680-46da7280-80a2-11ea-8a0a-7e33e6ef87dc.png)

(just noting this in case anyone else has seen this and has an idea)